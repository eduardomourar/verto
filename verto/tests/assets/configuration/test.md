{image file-path="http://placehold.it/350x150" caption="true" source="https://placehold.it/" hover-text="This is hover text" alignment="left" alt="This is alt text"}

Dancing with our hands tied.

{image end}

{image file-path="http://placehold.it/250x50" caption="false" source="https://placehold.it/" hover-text="This is hover text" alignment="left" alt="This is alt text"}
