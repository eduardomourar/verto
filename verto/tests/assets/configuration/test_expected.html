<div>
<img alt="This is alt text" class="left-align" src="http://placehold.it/350x150" title="This is hover text" />
<p>Dancing with our hands tied.</p>
<p><a href="https://placehold.it/">Source</a></p>
</div>
<div>
<img alt="This is alt text" class="left-align" src="http://placehold.it/250x50" title="This is hover text" />
<p><a href="https://placehold.it/">Source</a></p>
</div>
